bin/milano/deploy

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
This is all Power specific. You should remove it from the PR which you submit upstream.